packages/applicant/src/pages/api/aws-local.page.tsx

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { APIGlobalHandler } from '../../utils/apiErrorHandler';
+import { fromIni } from '@aws-sdk/credential-providers';
+import { IS_PRODUCTION } from '../../utils/constants';
+
+const credentials = fromIni({ profile: 'default' });
+
+// this handler is required to pull shared AWS credentials stored locally
+// for use in middleware, as the middleware cannot access the filesystem
+// directly
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (IS_PRODUCTION) return res.status(403).json('local dev use only');
```

Comment:
Will this also return a 403 in sandbox & QA?

Reply:
yep - this is env agnostic, it's a variable nextjs sets depending on whether you're running in dev or production mode. so any built instance of this app will return a 403, I should add a note to the readme that you'll need to edit this flag to run the app in production mode locally.

Reply:
awesome